mibidata/runs.py

```diff
@@ -41,8 +41,9 @@ def parse_xml(path):
             match = re.match(FOV_PATTERN, item.tag)
             if item.tag.startswith('RowNumber'):
                 row_num = item.tag
-                coordinates = (int(item.attrib.get('XAttrib')),
-                               int(item.attrib.get('YAttrib')))
+                coordinates = (
+                    float(item.attrib.get('XAttrib')) * _MOTOR_STEP,
+                    float(item.attrib.get('YAttrib')) * _MOTOR_STEP)
```

Comment:
If instead of `_MOTOR_STEP_` this were named `_MICRONS_PER_MOTOR_STEP` it'd be really obvious what this operation is doing.

Reply:
done